src/components/SearchBar/SearchBar.stories.tsx

```diff
@@ -86,6 +86,7 @@ export const WithControls: Story = {
         {...rest}
         onSubmit={(e) => {
           e.preventDefault();
+          console.log("Submitted!");
```

Comment:
Added this to confirm to myself that the Enter key and Search button are actually doing something, but as discussed [here](https://github.com/NYPL/nypl-design-system/pull/1597#discussion_r1603967244), I'm not really asserting that this will be logged, so it's sort of pointless

Reply:
This is a pattern that we have used elsewhere, so I think this is appropriate.